### PR TITLE
Include FT views in hot topics

### PR DESCRIPTION
**Pull request by i-like-robots**
screen shot 2015-07-17 at 15 13 04

### CODE REVIEW

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-17/14:22-2288/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-17/14:22-2288/successes/index.html)

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-07-17/14:52-2291/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-07-17/14:52-2291/successes/index.html)

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-07-17/14:56-2293/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-07-17/14:56-2293/successes/index.html)

**Comment by matthew-andrews** [comment]
:ship:

File: client/components/stream/_hot-topics.scss [hunk 1]
```diff
@@ -116,4 +116,13 @@
 	@include oGridRespondTo(L) {
 		@include nTypeAlpha(3);
 	}
+
+	.hot-topics__list-item--comment & {
+		@include nTypeCharlie(4);
+		@include nLinksHeadline(article-comment);
+
+		@include oGridRespondTo(L) {
+			@include nTypeCharlie(3);
```
Comment (keirog) [inline client/components/stream/_hot-topics.scss]: You could use `nTypeCharlieSize(3)` here

**Pull request merged**